docs/contributing/pages/redirects.mdx

```diff
@@ -3,31 +3,57 @@ title: Redirects
 noindex: true
 ---
 
-
 Redirects allow you to automatically redirect an incoming request path to a new destination path. When you move or rename a file, you should make sure to set up a redirect from the old path to the new path, so that the old URL still takes users to the right place.
 
 ## Add a New Redirect
 
-Redirects are configured in [`next.config.js`](https://github.com/getsentry/sentry-docs/blob/master/next.config.js) and are checked before the filesystem (including pages and `/public` files ). The `redirects` function returns an array holding redirect objects.
+There are two ways to add redirects in the Sentry docs:
+
+- [Add a New Redirect in `middleware.ts` (Recommended)](#add-a-new-redirect-in-middlewarets-recommended)
+- [Add a New Redirect in `vercel.json`](#add-a-new-redirect-in-verceljson)
+
+Because Sentry has a limited number of Vercel redirects, you should configure your redirects in `middleware.ts` whenever possible. You should only use `vercel.json` if you need to use regular expressions in your redirects.
+
+### Add a New Redirect in `middleware.ts` (Recommended)
+
+For simple one-to-one redirects, set them up in `src/middleware.ts`.
+
+To add a new redirect in [`src/middleware.ts`](https://github.com/getsentry/sentry-docs/blob/master/src/middleware.ts#L60), add a new object to `REDIRECTS` with the following properties:
+
+- `from`: The incoming request path.
+- `to`: The new destination path you want to route to instead.
+
+#### Example Redirect in `middleware.ts`
 
-To add a new redirect, add a new object in the redirects array with the following properties:
+The example below redirects `https://docs.sentry.io/performance/` to `https://docs.sentry.io/product/performance/`:
+
+```typescript {filename:middleware.ts}
```

Comment:
Let's add highlighting to the code block meta data:
```
```typescript {filename:middleware.ts} {2-5}
````